hangman/hangman/game.py

```
@@ -0,0 +1,65 @@
+
+    def run(self) -> None:
+        """Start the main game loop."""
+        print("H A N G M A N")
+        while True:
+            command = get_menu_command()
+            if command == MenuOption.EXIT:
+                break
+            elif command == MenuOption.PLAY:
+                self._play_one_game()
+            elif command == MenuOption.RESULTS:
+                self._show_results()
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "break", remove the leading "el" from "elif"

(R1723)